modules/apps/jaggery/emm/src/emm/units/role-listing/public/js/role-listing.js

```diff
@@ -131,7 +132,7 @@ $("#role-grid").on("click", ".remove-role-link", function () {
 
 $("#search-btn").click(function () {
     var searchQuery = $("#search-by-name").val();
-    if( ! searchQuery.trim() == "" ){
+    if (!searchQuery.trim() == "") {
```

Comment:
Noted.
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158